skaffold.yaml

```diff
@@ -1167,15 +1172,19 @@ profiles:
     - -ec
     - |
       echo "Wait until seed is ready"
-      for i in `seq 1 30`;
-      do
-        if kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG get seed local2 2> /dev/null; then
+      for _ in $(seq 1 60); do
+        seed_name="local2"
+
+        if [ "$(kubectl get seed "$seed_name" -o jsonpath='{.status.conditions[?(@.type=="GardenletReady")].status}')" == "True" ] &&
+          [ "$(kubectl get seed "$seed_name" -o jsonpath='{.status.conditions[?(@.type=="Bootstrapped")].status}')" == "True" ] &&
+          [ "$(kubectl get seed "$seed_name" -o jsonpath='{.status.conditions[?(@.type=="ExtensionsReady")].status}')" == "True" ]; then
```

Comment:
Here (https://github.com/gardener/gardener/pull/7439/files#diff-cb88f836d7d2eec7bccbed73c47a452b8eb3e376b02bae73ecae449bba96214aR1126-R1130, you are checking more conditions. Why not here as well?